src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java

```diff
@@ -74,7 +78,9 @@ public static File convert(File inputFile) throws TelegramApiException {
 
 			var image = toImage(inputFile);
 			if (image != null) {
-				return convertToPng(image, mimeType);
+				boolean isFileSizeCompliant = isFileSizeLowerThan(inputFile, MAX_IMAGE_FILE_SIZE);
```

Comment:
var?

Reply:
I'm not really a fan of var in place of primitive types